docs/browser-compatibility.md

```diff
@@ -0,0 +1,8 @@
+# Browser compatibility
+
+The Reaction Commerce component library aims to support all browsers supported by React, v16, outlined [here](https://reactjs.org/docs/react-dom.html#browser-support).
+
+Browser list:
+- Google Chrome
+- Mozilla Firefox
+- Internet Explorer 11 (IE 9 and 10 requires [using polyfills](https://reactjs.org/blog/2017/09/26/react-v16.0.html#javascript-environment-requirements))
```

Comment:
I feel like we should only outline support for IE 11, I don't see us actually trying to support those older versions of IE at this point.

Reply:
Completely agree. React can support IE9 and 10. I don't want to support those older versions of IE.

Reply:
Got it. Will remove mention of 9 and 10.